Mlem/API/Models/Site/APILocalSite.swift

```diff
@@ -9,36 +9,36 @@ import Foundation
 
 // lemmy_db_schema::source::local_site::LocalSite
 struct APILocalSite: Decodable {
-    let id: Int
-    let siteId: Int
-    let siteSetup: Bool
+//    let id: Int
+//    let siteId: Int
+//    let siteSetup: Bool
     let enableDownvotes: Bool
-    let enableNsfw: Bool
```

Comment:
keep this one, please

Reply:
The plan is to bring this whole model back but as a separate PR that avoids downstream areas depending on values it's not using.

Currently the only place we call this is to retrieve the single `enableDownvotes` value, in the meantime mapping the values means although they are unused the modelling will fail if any instance does not return them 😞

Reply:
ah, fair enough then